File: JavaScript/Basics/0.3-Conditional-Steatements-Advanced/Exercise/summer-outfit.js
function summerOutfit (input) {
    let degrees = Number(input[0]);
    let dayTime = input[1];
    let outfit, shoes;
    if (dayTime == "Morning") {
        if (10 <= degrees && degrees <= 18) {
            outfit = "Sweatshirt";
            shoes = "Sneakers";
        } else if (18 < degrees && degrees <= 24) {
            outfit = "Shirt";
            shoes = "Moccasins";
        } else if (degrees >= 25) {
            outfit = "T-Shirt";
            shoes = "Sandals";
        }
    } else if (dayTime == "Afternoon") {
        if (10 <= degrees && degrees <= 18) {
            outfit = "Shirt";
            shoes = "Moccasins";
        } else if (18 < degrees && degrees <= 24) {
            outfit = "T-Shirt";
            shoes = "Sandals";
        } else if (degrees >= 25) {
            outfit = "Swim Suit";
            shoes = "Barefoot";
        }
    } else if (dayTime == "Evening") {
        outfit = "Shirt";
        shoes = "Moccasins";
    }

    console.log(`It's ${degrees} degrees, get your ${outfit} and ${shoes}.`);
}

File: Python/0.1-Basics/0.3-Conditional-Statements-Advanced/Exercise/summer_outfit.py
degrees = int(input())
day_time = input()

outfit = ["Sweatshirt", "Shirt", "T-Shirt", "Swim Suit"]
shoes = ["Sneakers", "Moccasins", "Sandals", "Barefoot"]

if 10 <= degrees <= 18:
    if day_time == "Morning":
        outfit = outfit[0]
        shoes = shoes[0]
    elif day_time == "Afternoon":
        outfit = outfit[1]
        shoes = shoes[1]
    elif day_time == "Evening":
        outfit = outfit[1]
        shoes = shoes[1]
elif 18 < degrees <= 24:
    if day_time == "Morning":
        outfit = outfit[1]
        shoes = shoes[1]
    elif day_time == "Afternoon":
        outfit = outfit[2]
        shoes = shoes[2]
    elif day_time == "Evening":
        outfit = outfit[1]
        shoes = shoes[1]
elif degrees >= 25:
    if day_time == "Morning":
        outfit = outfit[2]
        shoes = shoes[2]
    elif day_time == "Afternoon":
        outfit = outfit[3]
        shoes = shoes[3]
    elif day_time == "Evening":
        outfit = outfit[1]
        shoes = shoes[1]

print(f"It's {degrees} degrees, get your {outfit} and {shoes}.")
